core/server/config/env/config.development.json

```diff
@@ -8,5 +8,16 @@
     },
     "paths": {
         "contentPath": "content/"
+    },
+    "privacy": {
+        "useRpcPing": false,
+        "useUpdateCheck": true
+    },
+    "minifyAssets": false,
```

Comment:
I think this might be misleading - it doesn't actually minify anything, just changes the file name to .min.whatever I think haha